gridstatus/base.py

```diff
@@ -101,8 +101,12 @@ def _latest_lmp_from_today(self, market, locations, **kwargs):
             date="today", market=market, locations=locations, **kwargs
         )
         col_order = lmp_df.columns.tolist()
+
+        # Special case to handle PJM 5 min LMPs
```

Comment:
This was only needed for PJM `Markets.REAL_TIME_5_MIN` data.

Reply:
Maybe this is related to the breaking change here: https://github.com/kmax12/gridstatus/blob/main/CHANGELOG.md#breaking-changes

Reply:
yep